app/schema.py

```diff
@@ -24,7 +24,31 @@ class Post(BaseModel):
     desc: str
     long: Decimal
     lat: Decimal
-    images: List[Images]
+    # images: List[Images]
```

Comment:
why class images has images list

Reply:
I think that might be a typo in the class name: should be singular (`Image`) and not plural (`Images`). Other than that, seems ok.

Reply:
I think that might be a typo in the class name: should be singular (Image) and not plural (Images). Other than that, seems ok.

Reply:
thats @n2oneProgrammer mistake

Reply:
I did't wrote that.